lib/data/wrapper.js

```diff
@@ -323,19 +379,22 @@ const data = {
      * @param {BackendInfo} destBackendInfo - Instance of BackendInfo:
      * Represents the info necessary to evaluate which data backend to use
      * on a data put call.
-     * @param {object} serverSideEncryption - serverSideEncryption
+     * @param {object} sourceBucketMD - metadata of the source bucket
+     * @param {object} destBucketMD - metadata of the destination bucket
      * @param {object} log - Werelogs request logger
      * @param {function} cb - callback
      * @returns {function} cb - callback
      */
     copyObject: (request,
       sourceLocationConstraintName, storeMetadataParams, dataLocator,
-      dataStoreContext, destBackendInfo, serverSideEncryption, log, cb) => {
+      dataStoreContext, destBackendInfo, sourceBucketMD, destBucketMD, log,
+      cb) => {
+        const serverSideEncryption = destBucketMD.getServerSideEncryption();
         if (config.backends.data === 'multiple') {
```

Comment:
you could reduce the amount of nesting if you get the destLocationConstraintName before and do:
```
if (config.backends.data === 'multiple' && utils.externalBackendCopy(sourceLocationConstraintName,
destLocationConstraintName, sourceBucketMd, destBucketMD)) {
...
}
```
or pass `config.backends.data` to `utils.externalBackendCopy` to be included in the check